### DISCUSSION TITLE
Problem saving table

### DISCUSSION THREAD

**Discussion by tobiassaueressig** [davidgohel/flextable#464] [Q&A] [answered]
Dear David,

thank you for providing this exceptional package! 

I have encountered the following problem: 

I have a table with around 750 entries 13 columns. I can create a flextable object and also view it in RStudio. I can also save it e.g. as an excel file. 

If I try to save it with save_as_docx() (or any other helper function) it prints an error: 

Error in read_xml.raw(charToRaw(enc2utf8(x)), "UTF-8", ..., as_html = as_html,  : 
  internal error: Huge input lookup [1]

Is there a possibility to work around this?

Regards,

Tobias Saueresssig

**Comment by davidgohel** [comment, marked as answer]
It's difficult to reproduce but it certainly happens because the data is too bit. flextable's goal is to enable static reporting, for reasonable table sizes. In your case I think a simple use of kable in an R Markdown document would be better.

**Reply by tobiassaueressig** [reply]
Thank you David for taking the time to reply!  

Best regards,
Tobias

**Comment by tobiassaueressig** [comment]
Hi David,

just wanted to say that I could output it via flextable by using Quatro and rendering to word. Thanks.
